elb_log_ingestor/elb_log_parse.py

```diff
@@ -112,8 +149,12 @@ def timestamp_to_timestamp(timestamp: str) -> str:
     "request_creation_time": timestamp_to_timestamp,
     "actions_executed": str,
     "redirect_url": str,
-    "lambda_error_reason": str,
+    "error_reason": str,
     "new_field": str,
+    "processing_targets_list": str,
+    "target_status_code_list": str,
+    "desync_mitigation_classification": str,
+    "desync_mitigation_classification_reason": str,
```

Comment:
we aren't indexing these currently.
`processing_targets_list` and `target_status_code_list` feel like they should be arrays, but I think elasticsearch doesn't guarantee array ordering and these are parallel, so proceed with caution